cmd/mesh/testdata/manifest-migrate/output/values.yaml

```diff
@@ -16,11 +16,6 @@ spec:
               maxUnavailable: 25%
           tolerations: []
     enabled: true
-  cni:
```

Comment:
Looks like we only have one golden test file - could you please also add at least Alex's config from the bug to the golden test set?

Reply:
I think this case is already covered in existing test input, for example there is no base component enablement  defined in input values.yaml thus no enablement for that defined in output ICPS.

Reply:
and also the unit test covered it

Reply:
I know, but this would be more of a sanity check so that we can be sure a small input file produces a small output file.

Reply:
added now